ipatests/test_integration/test_external_ca.py

```diff
@@ -17,11 +17,16 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import os
+import base64
 
 from ipatests.pytest_plugins.integration import tasks
 from ipatests.test_integration.base import IntegrationTest
 
 
+AKI_ID = base64.b64encode(os.urandom(64))
+SKI_ID = base64.b64encode(os.urandom(64))
```

Comment:
I think some clearer names would be helpful to people because e.g. we have
`AKI_ID` being used as a subject key ID below (for the external CA).

Perhaps `EXTERNAL_CA_KEY_ID` and `IPA_CA_KEY_ID` would be better names?